emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -369,7 +371,7 @@ def insert(self, entry):
         """
         Inserts the specified entry and returns the object ID 
         """
-        logging.debug("insert called with entry of type %s" % type(entry))
+        logging.info("insert called with entry of type %s" % type(entry))
```

Comment:
please revert, no need to have log spew at the `INFO` log level

Reply:
Understood. I was just using this to confirm it was working at the time

Is there a flag or option we can set to temporarily show debug logs, so we don't have to use `.info`?

Reply:
Resolved in https://github.com/shankari/e-mission-server/pull/4/commits/3fbc72d73cd4d0e73f5ceb336e8c151e2da1ebc9

Reply:
@JGreenlee the logging in general is controlled by files in `conf/log`. What you see on the console is not the full extent of the logging.